src/Angor/Client/Pages/Wallet.razor

```diff
@@ -305,8 +312,8 @@
                         </tr>
                     </thead>
                     <tbody>
-                    
-                        @foreach (var addressInfo in localAccountInfo.AddressesInfo.Union(localAccountInfo.ChangeAddressesInfo))
+
+                        @foreach (var addressInfo in accountBalanceInfo.AccountInfo.AddressesInfo.Union(accountBalanceInfo.AccountInfo.ChangeAddressesInfo))
```

Comment:
Would be easier if it was a method on the account balance info

Reply:
good point I will add that

Reply:
this should not even be Union it should be a concat I think